nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryRecord.java

```diff
@@ -312,6 +317,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                     final QueryResult queryResult = query(session, original, readerSchema, sql, recordReaderFactory);
 
                     final AtomicReference<String> mimeTypeRef = new AtomicReference<>();
+                    final FlowFile originalFlowFile = original;
```

Comment:
is this required?

Reply:
Yes I need a final variable an the inner class so I don't get this error: `Variable 'original' is accessed from within inner class, needs to be final or effectively final`.

The same was done for the AbstractRecordProcessor class [here](https://github.com/apache/nifi/blob/main/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractRecordProcessor.java#L119).